installation/resources/installer-config-cluster.yaml.tpl

```diff
@@ -29,17 +29,13 @@ metadata:
   labels:
     installer: overrides
 data:
-  global.isLocalEnv: "false"
-  global.knative: "false"
```

Comment:
This override needs to stay. Knative installation procedure relies on its existance.

Reply:
this will be covered in separate PR, linked to this issue: https://github.com/kyma-project/kyma/pull/2394